scripts/build_tools.xml

```diff
@@ -27,7 +27,14 @@
 		<mkdir dir="${build}" />
 	</target>
 
-	<target name="compile" depends="init" description="Using java ${JDK_VERSION} to compile the source  ">
+	<target name="getDependency" depends="init" description="get third party dependency">
+		<subant target="build">
+			<property name="LIB" value="json_simple"/>
+			<fileset dir="./" includes="getDependencies.xml" />
+		</subant>
+	</target>
```

Comment:
If we use `<import file="./getDependencies.xml"/>`, I think we can avoid defining the above target.